### ISSUE TITLE
Why need to get the mean value of the batch dimension?

### ISSUE THREAD

**Issue by xianxuhua** [jindongwang/Pytorch-CapsuleNet#16]
```
b_ij = b_ij + a_ij.squeeze(4).mean(dim=0, keepdim=True)
```

**Comment by NeVerDieKinG** [comment]
i guess it is similar to effect of Batch Normalization?
